R/dv8.R

```diff
@@ -0,0 +1,73 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#' Exports DV8 clustering (.dv8-clsx) as a separate JSON file
+#'
+#' @param dv8_path path to dv8 binary
+#' @param clustering_path path to clsx to convert
+#' @param output_path path to save output file
+#' @export
+dv8_clsxb_to_clsxj <- function(dv8_path, clustering_path, output_path){
+  system2(dv8_path, args=c('core:export-cluster', '-outputFile', output_path, clustering_path), stdout=FALSE, stderr=FALSE)
+}
+#' Parse a clustering json from DV8
+#'
+#' @param clustering_path path to clustering json (obtained from dv8_clsxb_to_clsxj)
```

Comment:
There is a specific syntax you can use here. Search the git repo for the syntax:

#' @seealso \code{\link{parse_gitlog}}

And add below the @family parsers line. This will significantly facilitate code navigation of the pipeline via the R docs function, since they hyperlink on R Studio. Please apply it as needed to the other function docs.

Reply:
I had the @family parsers line before I believe.